### ISSUE TITLE
Cycle.js as a strict dependency

### ISSUE THREAD

**Issue by staltz** [erykpiast/cyclejs-group#5]
```
  "dependencies": {
    "babel": "^5.2.6",
    "cyclejs": "^0.20.4",
    "foreach": "^2.0.5",
    "get-parameter-names": "^0.2.0",
    "map-values": "^1.0.1",
    "merge-object": "^1.0.0"
  },
```

If you have `cyclejs` as a dependency, then `cyclejs-group` is not an addon, it's like a wrapper to Cycle. This is a problem as a structure for addons. What if you would want to include addons `cyclejs-first`, `cyclejs-second`, `cyclejs-third`, and each of these would wrap slightly different versions of Cycle.js?
That's why I'd rather `require()` on run-time, not on module import time, and specially not on `npm install`.

**Comment by erykpiast** [comment]
I was pondering about it, but because I can't find straightforward solution with ES6 `import`, I decided to solve this issue later. What do you think about moving Cycle to NPM's `peerDependencies`?

**Comment by staltz** [comment]
I didn't know about peerDependencies. Good pointer. Should we try to use that?

**Comment by erykpiast** [comment]
I think we can try. With `peerDependencies` outcome is what we expect - `cyclejs` package is installed only in top-level `node_modules`, there is no copy of it in `cyclejs-group/node_modules`. But still everything works :)

Fixed with [0.3.4](https://github.com/erykpiast/cyclejs-group/releases/tag/v0.3.4).
